cli/Cargo.toml

```diff
@@ -24,15 +24,15 @@ axum = "0.7.4"
 base64 = "0.22.1"
 bytes = "1.1.0"
 chrono = { version = "^0.4", default-features = false, features = ["serde", "clock"] }
-cidr = "0.2.0"
+cidr = "0.3.0"
 clap = { version = "4.0.9", features = ["string", "wrap_help"] }
 console = "0.15.2"
 dashmap = "6.0.1"
 deno_ast = { version = "0.41.2", features = ["transpiling"], optional = true }
 deno_core = { version = "0.307.0", optional = true }
 deno_runtime = { version = "0.177.0", optional = true }
```

Comment:
Is it possible to bump these as well? It *might* [make DBot happy](https://github.com/phylum-dev/cli/security/dependabot/33).

Reply:
It's possible with some code changes that I didn't really want to get into at the moment.

But it wouldn't fix the `idna` issue because we have more paths to that crate. For example:

```
phylum-cli
└── git2 v0.19.0
    └── url v2.4.1
        └── idna v0.4.0
```

Reply:
Roger that...a separate PR then.